packages/stark-ui/src/modules/session-ui/components/session-card/session-card.component.ts

```diff
@@ -21,5 +21,5 @@ export class StarkSessionCardComponent {
 	 * The title shown in the header of the card
 	 */
 	@Input()
-	public cardTitle: string;
+	public cardTitle?: string;
```

Comment:
Why don't we initialize it as empty string "" as the other cases similar to this one?